azure-pipelines.yml

```diff
@@ -1,174 +1,189 @@
 pr:
   branches:
     include:
-    - dev
-    - release/*
+      - dev
+      - v*.x
 
 trigger:
   branches:
     include:
-    - dev
-    - release/*
-    
+      - dev
+      - v*.x
+  tags:
+    include:
+      - v*.*.*
+
+variables:
+  isTagTriggered: $[startsWith(variables['Build.SourceBranch'], 'refs/tags/')]
+  prefix: $[format('0.{0:yyyy}.{0:MMdd}', pipeline.startTime)]
+  version: $[format('{0}.{1}', variables.prefix, counter(variables.prefix, 1))] # e.g. 0.2001.0203.4
+
+pool:
+  name: 1ES-Hosted-AzFunc
```

Comment:
Why are we using 1ES hosted pool ?

Reply:
This is for 'BI and Operations' requirement. [Internal link](https://teams.microsoft.com/l/message/19:9b731c2fb23c4e71bc4e69985849db84@thread.tacv2/1639419692056?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=ea4f3f8a-b62c-4159-a29a-587132c2c06b&parentMessageId=1639419692056&teamName=Cybersecurity%20Executive%20Order%20Compliance%20-%20Azure%20Functions&channelName=Announcements%20%F0%9F%93%A2&createdTime=1639419692056).